aten/src/ATen/native/quantized/cpu/qconv.cpp

```diff
@@ -1529,5 +1822,18 @@ TORCH_LIBRARY_IMPL(_quantized, QuantizedCPU, m) {
   m.impl(TORCH_SELECTIVE_NAME("_quantized::conv_transpose2d"),  QConvInt8<2, false>::run);
 }
 
+TORCH_LIBRARY_IMPL(quantized, MkldnnCPU, m) {
+  m.impl(TORCH_SELECTIVE_NAME("quantized::qconv1d_pt2e"), QConvPT2E<PostOps::NoPostOp>::run);
+  m.impl(TORCH_SELECTIVE_NAME("quantized::qconv2d_pt2e"), QConvPT2E<PostOps::NoPostOp>::run);
+  m.impl(TORCH_SELECTIVE_NAME("quantized::qconv3d_pt2e"), QConvPT2E<PostOps::NoPostOp>::run);
+  // PostOP ReLU
+  m.impl(TORCH_SELECTIVE_NAME("quantized::qconv2d_relu_pt2e"), QConvPT2E<PostOps::Relu>::run);
+  // PostOP Add
+  m.impl(TORCH_SELECTIVE_NAME("quantized::qconv2d_add_pt2e"), QConvPT2E<PostOps::Add>::run_add);
+  // PostOP AddReLU
+  m.impl(TORCH_SELECTIVE_NAME("quantized::qconv2d_add_relu_pt2e"), QConvPT2E<PostOps::AddRelu>::run_add);
```

Comment:
Would it make more sense to make the interface more general with "pointwise"?

Reply:
Changed as similar to the first comment.